src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/util/HttpCacheUtils.java

```diff
@@ -63,7 +63,7 @@ public static boolean canBeServedByCache(
             switch (name)
             {
                 case CACHE_CONTROL:
-                    return value.contains("no-cache");
+                    return value.contains("no-cache") || value.equals("max-age=0");
```

Comment:
this is very basic.  It would be better if you implement max-age="X" and check X, where 0 is no special case

Reply:
(maybe add more tests as needed??)